docs/pbc/all/tax-management/202400.0/base-shop/vertex/install-vertex.md

```diff
@@ -85,9 +92,12 @@ use Spryker\Zed\TaxApp\Communication\Plugin\Calculation\TaxAppCalculationPlugin;
         $pluginStack = [
             // ...
         
-            # Please put this plugin after all other tax calculation plugins or instead.
+            # This plugin should be put after other tax plugins, but before GrandTotalCalculatorPlugin.
+            # Suggested plugins order is shown.
         
+            new TaxTotalCalculatorPlugin(),
             new TaxAppCalculationPlugin(),
+            new GrandTotalCalculatorPlugin(),
```

Comment:
please add each plugin to `use` list, so it's clear from which module the plugin is.

Reply:
Do we really need `TaxTotalCalculatorPlugin` from the default Spryker Tax? Because it's mentioned below this paragraph in the "Performance improvement" box. So will be confusing.

Reply:
> Do we really need TaxTotalCalculatorPlugin from the default Spryker Tax?

we are using that plugin for fallback if Vertex is not available or disabled

Reply:
OK, we use it in demoshops, but what if a customer wants to integrate Vertex only?

Reply:
The goal of adding the `TaxTotalCalculatorPlugin` to the example was so the reader would know where to put the new plugin, as it was otherwise just based on this description:
> Please put this plugin after all other tax calculation plugins or instead

Reply:
Please check this paragraph with blue background 
![image](https://github.com/spryker/spryker-docs/assets/454530/ce289b7a-7f5d-4877-9043-a2ea75971ab2)


If we recommend removing it we should **not** add it to the Plugins list as an example, if it's still required, this plugin has to be excluded from the "Performance improvement" section.

Reply:
it can be removed
this depends entirely on whether we want to provide the customer with a fallback to OOTB tax in case Vertex is broken/unresponsive/disabled via the "isActive" switch or not.

I can suggest that we change the highlighted line to the following:
`To enhance performance when using an external Tax calculation provider, you can disable the following plugins:`